codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2FrameReader.java

```diff
@@ -566,6 +575,9 @@ private static void readGoAwayFrame(ChannelHandlerContext ctx, ByteBuf payload,
 
     private void readWindowUpdateFrame(ChannelHandlerContext ctx, ByteBuf payload,
             Http2FrameListener listener) throws Http2Exception {
+        if (payloadLength != 4) {
```

Comment:
`Http2CodecUtil.WINDOW_UPDATE_FRAME_LENGTH` exists and uses `INT_FIELD_LENGTH`. Consider using a similar define here or making this check use a common static final variable.